cmsplugin_profile/templates/admin/profile/profile_form.html

```diff
@@ -1,10 +1,13 @@
 {% with instance=form.instance %}
-<div class="ui-widget inline-related{% if forloop.last %} empty-form last-related{% endif %}{% if instance.thumbnail_image.url %} complete {%endif%}" id="{{ form.prefix }}{% if forloop.last %}-empty{% endif %}" {% if instance.thumbnail_image.url %}style="background: #efefef url('{{ instance.thumbnail_image.url }}') no-repeat center;background-size: contain;"{% endif %}>
+<div class="ui-widget inline-related{% if forloop.last %} empty-form last-related{% endif %}{% if instance.thumbnail_image.url %} complete {%endif%}{% if form.validation_errors.delete_alone or form.validation_errors.delete_with_other %} has-errors{% endif %}"
+	id="{{ form.prefix }}{% if forloop.last %}-empty{% endif %}" {% if instance.thumbnail_image.url %}style="background: #efefef url('{{ instance.thumbnail_image.url }}') no-repeat center;background-size: contain;"{% endif %}>
 	<div class="profile-item-actions">
 		<a class="edit-profile-item" data-profile-id-prefix="{{form.prefix}}" href="#"><span class="inline-edit ace-icon fa fa-pencil" unselectable="on"></span></a>
-		<a class="delete-profile-item" data-profile-id="{{instance.id}}"
-		data-profile-id-prefix="{{form.prefix}}" href="#">
-		<span class="inline-edit ace-icon fa fa-times" unselectable="on"></span>
+		<a class="delete-profile-item" data-profile-id="{{instance.id}}" data-profile-id-prefix="{{form.prefix}}" href="#"><span class="inline-edit ace-icon fa fa-times" unselectable="on"></span>
+		</a>
+		{% if form.validation_errors.delete_alone or form.validation_errors.delete_with_other %}
+		<span class="message-profile-item" data-profile-id="21" data-profile-id-prefix="profile_set-0" data-rel="popover" data-trigger="click" data-placement="auto" data-content='<ul class="profile-errors list-unstyled">{% if form.validation_errors.delete_alone %}<li>Deleting profile would prevent the following Promo Grid(s) from displaying:<ul>{% for promo in form.validation_errors.delete_alone %}<li>Go To: <a href="{% url 'admin:cms_page_change' promo.page.id %}" target="_blank">{{ promo.short_info }}</a></li>{% endfor %}</ul></li>{% endif %}{% if form.validation_errors.delete_with_other %}<li>Deleting profile (along with others) would prevent the following Promo Grid(s) from displaying:<ul>{% for promo in form.validation_errors.delete_with_other %}<li>Go To: <a href="{% url 'admin:cms_page_change' promo.page.id %}" target="_blank">{{ promo.short_info }}</a></li>{% endfor %}</ul></li>{% endif %}</ul>' data-original-title="" data-html="true" title="">Click to see details.</span>
+		{% endif %}
```

Comment:
`data-content`: can this be moved in a hidden div maybe and have it formatted? It's really hard to read